RecoTauTag/RecoTau/python/PATTauDiscriminantCutMultiplexer_cfi.py

```diff
@@ -23,6 +23,11 @@
         ),
     key = cms.InputTag("fixme"), # a discriminator
     loadMVAfromDB = cms.bool(True),
+    inputFileName = cms.FileInPath("RecoTauTag/RecoTau/test/dummyMVAinputFile"), # the filename for MVA if it is not loaded from DB
```

Comment:
Isn't there a better solution rather than making all these configurations depend on an empty file based in a ```\test``` area?

I suspect it will throw anyhow even with this dummy and empty input file, see e.g. https://github.com/cms-sw/cmssw/blob/a33f80ee555411dc175cf67f519313b6644d2878/RecoTauTag/RecoTau/plugins/PFRecoTauDiscriminationAgainstMuonMVA.cc#L48 
Then, why not to simply keep ```""``` as the default, and remove the empty file ```RecoTauTag/RecoTau/test/dummyMVAinputFile```?

Reply:
Yes, sounds reasonable. To be changed